dropshot/src/router.rs

```diff
@@ -180,10 +181,38 @@ impl<'a> From<&'a str> for InputPath<'a> {
     }
 }
 
+static INFORM_PROC_MACRO2_WE_ARE_NOT_A_PROC_MACRO: Once = Once::new();
+
 /*
  * Validate that the string is a valid Rust identifier.
  */
 fn valid_identifier(var: &str) -> bool {
+    // TODO: Remove this "Once" callback when the following issue
```

Comment:
FWIW @davepacheco has tried (and failed) to hold the line on /* */ comments in dropshot so... do what you will with that .

Reply:
Interesting, I always considered that comment style "valid, but non-idiomatic" - the Rust book only shows the "//" style (see: [here](https://doc.rust-lang.org/stable/book/ch03-04-comments.html))  and there was even an (admittedly unsuccessful) attempt to remove block comments: https://github.com/rust-lang/rfcs/issues/1371

(Plus I [heard a rustlang maintainer also had the spicy take that they're not idiomatic](https://github.com/rust-lang/book/issues/693#issuecomment-301200629))

Do you want me to switch before landing? I noticed we're pretty inconsistent across many of Oxide's repos (dunno if we should clarify which repos do / don't have this styling more explicitly).

Reply:
[The rustdoc book](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html) also exclusively uses the non-block comment style ("//" or "///")

Reply:
I know.